### DISCUSSION TITLE
I think that key-bindings.md has incorrect file for `base_keymap` field

### DISCUSSION THREAD

**Discussion by amanzanero** [zed-industries/zed#11708] [Feature Requests]
# Problem

I was taking a look at key-bindings.md, specifically: https://github.com/zed-industries/zed/blob/48cba328f2235d8d2aa7126dd5d57c7e49fe13af/docs/src/key-bindings.md?plain=1#L15

since I wanted to change them to `JetBrains`. The docs say to set `base_keymap` in `keymap.json`, but I don't actually think that's a valid field? I was able to set the keymap in `settings.json` however.

My `settings.json`
```json
{
  "show_copilot_suggestions": true,
  "theme": "Solarized Light",
  "ui_font_size": 16,
  "buffer_font_size": 16,
  "base_keymap": "JetBrains"
}
```

# Proposal 

I think the docs should instead say
```diff
- These keymaps can be set via the `base_keymap` setting in your `keymap.json` file. Additionally, if you'd like to work from a clean slate, you can provide `"None"` to the setting.
+ These preset keymaps can be set via the `base_keymap` setting in your `settings.json` file. Additionally, if you'd like to work from a clean slate, you can set the value to `"None"` .
```

Happy to file a PR  :)

**Comment by Ahmard** [comment]
> # Problem
> 
> I was taking a look at key-bindings.md, specifically:
> 
> https://github.com/zed-industries/zed/blob/48cba328f2235d8d2aa7126dd5d57c7e49fe13af/docs/src/key-bindings.md?plain=1#L15
> 
> since I wanted to change them to `JetBrains`. The docs say to set `base_keymap` in `keymap.json`, but I don't actually think that's a valid field? I was able to set the keymap in `settings.json` however.
> 
> My `settings.json`
> 
> ```json
> {
>   "show_copilot_suggestions": true,
>   "theme": "Solarized Light",
>   "ui_font_size": 16,
>   "buffer_font_size": 16,
>   "base_keymap": "JetBrains"
> }
> ```
> 
> # Proposal
> 
> I think the docs should instead say
> 
> ```diff
> - These keymaps can be set via the `base_keymap` setting in your `keymap.json` file. Additionally, if you'd like to work from a clean slate, you can provide `"None"` to the setting.
> + These preset keymaps can be set via the `base_keymap` setting in your `settings.json` file. Additionally, if you'd like to work from a clean slate, you can set the value to `"None"` .
> ```
> 
> Happy to file a PR :)

Oh my God, I've been doing it the wrong way.
You're right, it should be specified.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
